### ISSUE TITLE
eslint does not satisfy its siblings' peerDependencies requirements

### ISSUE THREAD

**Issue by alexmngn** [eslint/eslint#5704]
**What version of ESLint are you using?** 2.4.0

**What parser (default, Babel-ESLint, etc.) are you using?** babel-eslint

**eslint config**
```
{
  "parser": "babel-eslint",
  "ecmaFeatures": {
    "classes": true,
    "jsx": true
  },
  "plugins": [
    "react"
  ],
  "env": {
    "jasmine": true
  },

  "extends": "eslint-config-airbnb"
}
```

**package.json dependencies for eslint**
```
{
  "devDependencies": {
    "babel-eslint": "^5.0.0",
    "eslint": "~2.4.0",
    "eslint-config-airbnb": "^6.1.0",
    "eslint-plugin-react": "^4.2.3",
  }
}
```

I get an error when try to install eslint on a fresh project:
```

npm ERR! Darwin 14.5.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "install"
npm ERR! node v4.3.2
npm ERR! npm  v2.14.12
npm ERR! code EPEERINVALID

npm ERR! peerinvalid The package eslint@2.4.0 does not satisfy its siblings' peerDependencies requirements!

```

I also tried with the latest version 2.5.1 but it doesn't change anything.

Any idea what's wrong? I looks like a dependency changed recently that breaks eslint from getting installed, as it was working a few days ago with exactly the same version.

**Comment by platinumazure** [comment]
Can you please share what ESLint shareable configurations, plugins, or other node modules you are using? ESLint doesn't declare any peer dependencies, so someone else has a peer dependency requirement which ESLint isn't satisfying.

**Comment by alexmngn** [comment]
@platinumazure Just updated the ticket with the eslint config file.

**Comment by ilyavolodin** [comment]
@alexmngn What were you doing when you run into this error? Just `npm install`? In that case, we would need to see your package.json file too.

**Comment by alexmngn** [comment]
@ilyavolodin yes just an `npm install`

**Comment by btmills** [comment]
babel-eslint v5 is for eslint v1. You should be able to upgrade to babel-eslint v6, which is no longer in beta, and has been updated for compatibility with eslint v2.

**Comment by alexmngn** [comment]
@btmills This fixed my issue and it's very weird! I was using v5 of babel for a month and it was working very well before something suddenly happened during this weekend...

**Comment by btmills** [comment]
@alexmngn glad I could help! Here's your culprit: https://github.com/babel/babel-eslint/commit/e29eedb037bab1b88cf44403aff9db0a1c4e400b. Released as 5.0.1 over the weekend.
